bazel/digest.go

```diff
@@ -0,0 +1,22 @@
+package bazel
+
+// Digest utilities for Bazel
+
+import (
+	"fmt"
+
+	remoteexecution "google.golang.org/genproto/googleapis/devtools/remoteexecution/v1test"
+)
+
+// Validate Digest hash and size components assuming SHA256
+func IsValidDigest(hash string, size int64) bool {
+	return len(hash) == 64 && size >= 0
```

Comment:
Would it be possible to drop the size >= 0 constraint, and remove size from the addressing information of blobs?

I have desires to fetch digests without knowing how big they are in advance, and it feels like passing size=-1 is a better way of doing that then specifying a valid-but-incorrect size.

(I will push this feedback upstream too)